serverless/security/serverless-custom-tls-cert-domain-mapping.adoc

```diff
@@ -13,59 +13,8 @@ You can use an existing TLS certificate with a `DomainMapping` custom resource (
 
 * You have completed the steps in xref:../../serverless/security/serverless-custom-domains.adoc#serverless-custom-domains[Configuring a custom domain for a Knative service], and have a working `DomainMapping` CR.
 
-* You have a TLS certificate from your Certificate Authority provider, or a self-signed certificate.
+* You have a TLS certificate from your Certificate Authority provider or a self-signed certificate.
 
 * You have obtained the `cert` and `key` files from your Certificate Authority provider, or a self-signed certificate.
 
-.Procedure
```

Comment:
Maybe check with @abrennan89, but if you're breaking this out into a module, I would have assumed you'd want the prerequisites above to go with it

Reply:
oh, right, it's because the prerequisites have an xref 

Is there a reason why we can't just leave this as it is to avoid confusion? I'm not really comfortable splitting the rest of the content away from the prereqs. I feel like this is one area that the not having xrefs in modules rule is an issue for us.

Reply:
@abrennan89 The current format makes the Ganymede build fail, so it needs to be restructured. Splitting the procedure into a procedure module fixes it. If you want the prereqs in the same file, I think it makes sense to duplicate the prereq that contains the xref and have it once in the assembly with the xref and once in the module without it. I'm pushing that change, so PTAL.

Reply:
@abrennan89, will you PTAL? https://deploy-preview-40403--osdocs.netlify.app/openshift-enterprise/latest/serverless/security/serverless-custom-tls-cert-domain-mapping.html

Reply:
lgtm!